src/app/loadout/loadout-ui/PlugDef.tsx

```diff
@@ -20,16 +19,14 @@ export default function PlugDef({ plug, onClick, onClose }: Props) {
   const showTooltip = defs && !isPhonePortrait;
 
   const contents = (
-    <div className={styles.emptyItem}>
```

Comment:
I can't see why this class was needed. Seems to behave as I would expect where it shows up still.